src/wallet/rpc/transactions.cpp

```diff
@@ -634,6 +636,14 @@ RPCHelpMan listsinceblock()
     bool include_removed = (request.params[3].isNull() || request.params[3].get_bool());
     bool include_change = (!request.params[4].isNull() && request.params[4].get_bool());
 
+    const std::string* filter_label = nullptr;
```

Comment:
use `std::optional` instead of nullptr

Reply:
Sounds good but is there a strong reason considering `ListTransactions`?

Reply:
from https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c-functions-and-methods
> Generally, use std::optional to represent optional by-value inputs

I think it's worth the refactor because pointers are not intended to be used as optional values.